### PR TITLE
Visor glasses and eye piece varients

### PR DESCRIPTION
**Pull request by Snamberr**
5 colors for the eye piece and visor glasses on the marine helmet.

### CODE REVIEW

File: src/Addons/34thPRC_AceArsenalExtended/config.cpp [hunk 1]
```diff
@@ -188,7 +188,7 @@ class XtdGearModels
 		{
 			label = "[34thPRC] CH252 Helmets";
 			author = "34th PRC Modding Team, J. Koko";
-			options[] = {"camo", "eyewear", "medic"};
+			options[] = {"camo", "eyewear", "medic", "color"};
```
Comment (Lupus590) [inline src/Addons/34thPRC_AceArsenalExtended/config.cpp:191]:
```suggestion
			options[] = {"camo", "eyewear", "medic", "eyewear_color"};

```

File: src/Addons/34thPRC_AceArsenalExtended/config.cpp [hunk 2]
```diff
@@ -247,6 +247,33 @@ class XtdGearModels
 					label = "Medic";
 				};
 			};
+			class color
```
Comment (Lupus590) [inline src/Addons/34thPRC_AceArsenalExtended/config.cpp:250]:
```suggestion
			class eyewear_color

```

File: src/Addons/34thPRC_AceArsenalExtended/config.cpp [hunk 3]
```diff
@@ -812,13 +839,80 @@ class XtdGearInfos
 			camo = "urban";
 			eyewear = "eyepiece";
 			medic = "false";
+			color = "green"
```
Comment (Lupus590) [inline src/Addons/34thPRC_AceArsenalExtended/config.cpp:842]:
```suggestion
			eyewear_color = "green"

```

**Review by Lupus590 — CHANGES_REQUESTED** [review]
color is a little too generic

**Comment by Lupus590** [comment]
Can you update the changelog too?

**Comment by Snamberr** [comment]
sure, do I put the changes in Unreleased or do i make a 0.17.0 in the changelog?

**Comment by Lupus590** [comment]
Colour bleed on tropic purple non-medic eyepeice
image

Otherwise it looks good

**Comment by Snamberr** [comment]
noticed another issue, gonna fix it

**Comment by Snamberr** [comment]
I don't see anymore issues in game.

**Review by Lupus590 — APPROVED** [review]

**Pull request merged**